gcn/notices/core/Localization.schema.json

```diff
@@ -13,25 +13,41 @@
       "type": "number",
       "description": "ICRS declination [deg], utilizes the J2000 epoch and an equatorial coordinate system"
     },
-    "uncertainty_shape": {
-      "enum": ["circle", "ellipse", "rectangle"],
-      "description": "In what manner are the errors in RA and Dec defined - as circular, elliptical, or rectangular shapes",
-      "default": "circle"
-    },
-    "ra_uncertainty": {
-      "type": "array",
-      "maxItems": 2,
-      "description": "Uncertainty in RA [deg] with optional asymmetric errors, measured from the RA in the schema; respective ex: [r], [r1], [l1-, l1+]. For ellipse, ra axis is r1 or semi-major axis."
-    },
-    "dec_uncertainty": {
-      "type": "array",
-      "maxItems": 2,
-      "description": "Uncertainty in Dec [deg] with optional asymmetric errors, measured from the Dec in the schema; respective ex: [r2], [l2-, l2+]. Not reported if circular and Dec axis is r2 or semi-minor axis for ellipse."
-    },
-    "position_angle": {
-      "type": "number",
-      "description": "Anti-clockwise position angle from RA axis [deg]. Used if uncertainty_shape ellipse or rectangle is tilted w.r.t. standard (RA, Dec) coordinate.",
-      "default": 0
+    "error": {
```

Comment:
'error' is a bit too generic, there can be error in many quantities.  We could call it position_error.  This nesting seems to break the flattening of the schema, and be a bit clunky.